### PR TITLE
Y24-353 Fix for Missing data on edit for ONT runs

### PR DESCRIPTION
**Pull request by seenanair**
Closes #

#### Changes proposed in this pull request

Update the 'include' parameter in the 'find' and 'create' method of createRequest so that destructuring of parameters works as in v1

Correct unit test cases to replicate the way it is called from stores

#### Instructions for Reviewers

_[All PRs] - Confirm PR template filled_  
_[Feature Branches] - Review code_  
_[Production Merges to `main`]_  
 &nbsp; &nbsp; \- _Check story numbers included_  
 &nbsp; &nbsp; \- _Check for debug code_  
 &nbsp; &nbsp; \- _Check version_

### CODE REVIEW

**Review by stevieing — CHANGES_REQUESTED** [review]
Do we need an addition to the e2e test for creating and editing an ONT run to ensure the data is shown? This will guard against a similar issue in the future.

**Comment by seenanair** [comment]
> Do we need an addition to the e2e test for creating and editing an ONT run to ensure the data is shown? This will guard against a similar issue in the future.

I actually tried it, but it didn't work as I expected. Might need bit more time looking into it.

**Comment by stevieing** [comment]
> > Do we need an addition to the e2e test for creating and editing an ONT run to ensure the data is shown? This will guard against a similar issue in the future.
> 
> I actually tried it, but it didn't work as I expected. Might need bit more time looking into it.

OK. Let's timebox it and if needed roll it into it's own story. It may well be an issue with the page loading. The ONT run page is very slow to load. We do have a technical debt story somewhere.

**Comment by seenanair** [comment]
> > > Do we need an addition to the e2e test for creating and editing an ONT run to ensure the data is shown? This will guard against a similar issue in the future.
> > 
> > 
> > I actually tried it, but it didn't work as I expected. Might need bit more time looking into it.
> 
> OK. Let's timebox it and if needed roll it into it's own story. It may well be an issue with the page loading. The ONT run page is very slow to load. We do have a technical debt story somewhere.

Ok. Also, it would be good to add similar e2e test cases for Pacbio Pool, Runs etc. for the same reason

**Comment by stevieing** [comment]
> > > > Do we need an addition to the e2e test for creating and editing an ONT run to ensure the data is shown? This will guard against a similar issue in the future.
> > > 
> > > 
> > > I actually tried it, but it didn't work as I expected. Might need bit more time looking into it.
> > 
> > 
> > OK. Let's timebox it and if needed roll it into it's own story. It may well be an issue with the page loading. The ONT run page is very slow to load. We do have a technical debt story somewhere.
> 
> Ok. Also, it would be good to add similar e2e test cases for Pacbio Pool, Runs etc. for the same reason

I have created a story  #1984

**Review by stevieing — DISMISSED** [review]

File: tests/e2e/specs/ont/ont_runs_view.cy.js [hunk 1]
```diff
@@ -23,4 +24,35 @@ describe('ONT Runs view', () => {
...
+    cy.intercept('/v1/ont/pools?include=tube,libraries.tag,libraries.request', {
+      fixture: 'tractionOntPools.json',
+    })
+
+    cy.visit('#/ont/runs')
+    cy.get('#editRun-2').click()
+
+    // Check that the URL is correct
+    cy.url().should('include', '#/ont/run/2')
+    cy.get('#flowcell-id-1').invoke('val').should('eq', 'ABC1234')
```
Comment (stevieing) [inline tests/e2e/specs/ont/ont_runs_view.cy.js:55]: Advisory. We should probably change this so it relates to the store data but I suggest we do that at another time and get this story deployed.

**Review by stevieing — APPROVED** [review]

File: tests/e2e/specs/ont/ont_runs_view.cy.js (hunk 1 above)
Comment (seenanair) [reply tests/e2e/specs/ont/ont_runs_view.cy.js:55]: Ok 👍

**Pull request merged**